.changeset/spotty-adults-leave.md

```diff
@@ -0,0 +1,7 @@
+---
+"graphql-manual": patch
+"@neo4j/graphql-toolbox": patch
+"@neo4j/graphql": patch
```

Comment:
```suggestion
"@neo4j/graphql-toolbox": patch
```

This should really just be a bump to the Toolbox, we don't version the docs right now and there's no change to the library here!